pallets/creditcoin/src/ocw/tests.rs

```diff
@@ -28,8 +28,8 @@ use crate::{
 		get_mock_amount, get_mock_contract, get_mock_from_address, get_mock_input_data,
 		get_mock_nonce, get_mock_timestamp, get_mock_to_address, get_mock_tx_block_num,
 		get_mock_tx_hash, roll_by_with_ocw, roll_to, roll_to_with_ocw, set_rpc_uri, AccountId,
-		Call, ExtBuilder, Extrinsic, MockedRpcRequests, Origin, PendingRequestExt, RwLock, Test,
-		ETHLESS_RESPONSES,
+		ExtBuilder, Extrinsic, MockedRpcRequests, PendingRequestExt, RuntimeCall as Call,
+		RuntimeOrigin as Origin, RwLock, Test, ETHLESS_RESPONSES,
```

Comment:
The thing is that I actually prefer `RuntimeOrigin` and `RuntimeCall` over just `Origin` and `Call` since it's clearer that it's generated for the overarching runtime (since every pallet also has its own `Origin` and `Call`). 

For the tests though, where they're repeated so many times, I'm fine with the shorter, less clear versions.